tools/gyp/.github/workflows/Python_tests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       max-parallel: 8
       matrix:
         os: [macos-latest, ubuntu-latest] # , windows-latest]
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Comment:
tools/gyp changes should be done in https://github.com/nodejs/gyp-next

Reply:
Testing with nodejs/gyp-next#127

Reply:
> tools/gyp changes should be done in https://github.com/nodejs/gyp-next

Thank you.